blueman/plugins/applet/StandardItems.py

```diff
@@ -79,53 +60,53 @@ def change_sensitivity(self, sensitive):
             power = True
 
         sensitive = sensitive and self.Applet.Manager and power
-        self.new_dev.props.sensitive = sensitive
-        self.send.props.sensitive = sensitive
-        self.devices.props.sensitive = sensitive
-        self.adapters.props.sensitive = sensitive
+        self.new_dev.set_sensitive(sensitive)
+        self.send.set_sensitive(sensitive)
+        self.devices.set_sensitive(sensitive)
+        self.adapters.set_sensitive(sensitive)
 
     def on_manager_state_changed(self, state):
         self.change_sensitivity(state)
 
     def on_power_state_changed(self, manager, state):
         self.change_sensitivity(state)
 
-    def on_setup_new(self, menu_item):
+    def on_setup_new(self):
         launch("blueman-assistant", None, False, "blueman", _("Bluetooth Assistant"))
 
-    def on_send(self, menu_item):
+    def on_send(self):
         launch("blueman-sendto", None, False, "blueman", _("File Sender"))
 
-    def on_devices(self, menu_item):
+    def on_devices(self):
         lockfile = get_lockfile('blueman-manager')
         pid = get_pid(lockfile)
         if lockfile and pid and is_running('blueman-manager', pid):
             os.kill(pid, signal.SIGTERM)
         else:
             launch("blueman-manager", None, False, "blueman", _("Device Manager"))
 
-    def on_adapters(self, menu_item):
+    def on_adapters(self):
         launch("blueman-adapters", None, False, "blueman", _("Adapter Preferences"))
 
-    def on_local_services(self, menu_item):
+    def on_local_services(self):
         launch("blueman-services", None, False, "blueman", _("Service Preferences"))
 
-    def on_about(self, menu_item):
+    def on_about(self):
         about = show_about_dialog("Blueman " + _("applet"), run=False)
 
         button = Gtk.Button(_("Plugins"))
         button.set_image(Gtk.Image.new_from_icon_name("blueman-plugin", Gtk.IconSize.BUTTON))
         button.show()
 
-        button.connect("clicked", self.on_plugins)
+        button.connect("clicked", lambda _button: self.on_plugins())
 
         about.action_area.pack_start(button, True, True, 0)
         about.action_area.reorder_child(button, 0)
 
         about.run()
         about.destroy()
 
-    def on_plugins(self, button):
+    def on_plugins(self):
```

Comment:
This broke the dbus method used by manager. We should drop ``None`` from [here](https://github.com/cschramm/blueman/blob/13381dc017e84b8dc816e8732681a5c52180286b/blueman/plugins/applet/DBusService.py#L127)